ONSdigital/florence
# Remove domain from recipe api proxy call

### What

Fixed a couple of issues around this.

1) JS was hard coded to call http://localhost:8081, but in an environment florence is unlikely to run on 8081, so removed this.
2) The modal would only load the last dataset from the recipe api, so it has been updated to show all.

### How to review

- Add an API dataset landing page to a collection
- Verify modal appears correctly and you can see all datasets from recipe api 

### Who can review

Anyone
